### PR TITLE
Release 3.37

### PR DESCRIPTION
**Pull request by abidlabs**

### CODE REVIEW

File: js/chatbot/src/static/ChatBot.svelte [hunk 1]
```diff
@@ -359,6 +360,7 @@
 	/* KaTeX */
 	.message-wrap :global(span.katex) {
 		font-size: var(--text-lg);
+		direction: ltr;
```
[the remark this answers is not in the document]
Comment (abidlabs) [reply js/chatbot/src/static/ChatBot.svelte:363]: Yup! That's where this screenshot is from (the last Chatbot story). Or did you mean something else?

[the remark this answers is not in the document]
Comment (freddyaboulton) [reply js/chatbot/src/static/ChatBot.svelte:363]: My bad- didn't see this

```svelte
<Story
	name="Chatbot with text rendered right-to-left"
	args={{ rtl: true, latex_delimiters: [{ "left": "$$", "right": "$$", "display": true }]}}
/>
```

[the remark this answers is not in the document]
Comment (abidlabs) [reply js/chatbot/src/static/ChatBot.svelte:363]: nw, thinking alike!

[the remark this answers is not in the document]
Comment (abidlabs) [reply js/chatbot/src/static/ChatBot.svelte:363]: Learned all this from @hannahblair and @pngwn's great review

**Comment by abidlabs** [comment]
Demos LGTM!

**Review by freddyaboulton — APPROVED** [review]
Looks good to me as well!!

**Pull request merged**